src/commonMain/kotlin/fr/acinq/lightning/payment/PaymentRequest.kt

```diff
@@ -282,13 +269,12 @@ data class PaymentRequest(
         )
     }
 
-    @Serializable
     sealed class TaggedField {
         abstract val tag: Int5
         abstract fun encode(): List<Int5>
 
         /** @param description a free-format string that will be included in the payment request */
-        @Serializable
+
```

Comment:
nit: the find-and-replace produced many empty lines between the documentation comment and its class in this file

Reply:
I confess I did a global replace 😇

Reply:
Did a global replace to fix my first global replace (yodawg.jpg).